get-cloudify.py

```diff
@@ -535,100 +552,276 @@ def check_cloudify_installed(virtualenv_path=None):
 
 def handle_upgrade(upgrade=False, virtualenv=''):
     if check_cloudify_installed(virtualenv):
-        lgr.info('Cloudify is already installed in the path.')
+        logger.info('Cloudify is already installed in the path.')
         if upgrade:
-            lgr.info('Upgrading...')
+            logger.info('Upgrading...')
         else:
-            lgr.error('Use the --upgrade flag to upgrade.')
+            logger.error('Use the --upgrade flag to upgrade.')
             sys.exit(1)
 
 
 def parse_args(args=None):
     class VerifySource(argparse.Action):
         def __call__(self, parser, args, values, option_string=None):
-            if not args.source:
-                parser.error(
-                    '--source is required when calling --withrequirements.')
+            if not args.source and not args.use_branch:
+                parser.error('--source or --use-branch is required when '
+                             'calling with --with-requirements.')
             setattr(args, self.dest, values)
 
     parser = argparse.ArgumentParser(
-        description=DESCRIPTION, formatter_class=argparse.RawTextHelpFormatter)
-    default_group = parser.add_mutually_exclusive_group()
+        description=DESCRIPTION,
+        formatter_class=argparse.RawTextHelpFormatter,
+    )
+
+    if IS_WIN:
+        python_path_default = 'c:/python27/python.exe'
```

Comment:
Um.. I think it's my bad.

I would not differentiate windows from linux here and just use `sys.executable`.